gear-rpc-client/src/lib.rs

```diff
@@ -688,4 +688,29 @@ impl GearApi {
 
         Ok(events.collect())
     }
+
+    /* todo(playX18): some version mismatch between relayer and this crate does not allow us to return AccountId32 as is, return raw bytes for now */
+    pub async fn bridge_admin(&self) -> anyhow::Result<[u8; 32]> {
+        let addr = subxt::dynamic::constant("GearEthBridge", "BridgeAdmin");
```

Comment:
Looks like we can extract a new function called, for example, `get_constant(pallet, name)` and use it here and in the `bridge_pauser` the following way:
```rust
    pub async fn bridge_admin(&self) -> anyhow::Result<[u8; 32]> {
        self.get_constant("GearEthBridge", "BridgeAdmin")
             .context("Failed to get BridgeAdmin address")
    }
```